docs/process/roadmap.md

```diff
@@ -2,29 +2,29 @@
 
 The following are the larger areas of upcoming work the GitHub Desktop team intends to explore. This is not inclusive of everything we're working on, and it's not written in stone. We'll continue to update it as our priorities evolve.
 
-#### Merge conflicts iteration
+#### Support rebase when pulling (in progress)
 
-- Iterate on initial merge conflicts ship: [#6213](https://github.com/desktop/desktop/issues/6213)
-- Measuring success: [#6388](https://github.com/desktop/desktop/issues/6388)
+- Respect user's git config for pull --rebase and resolve conflicts: [#3422](https://github.com/desktop/desktop/issues/3422)
+- Measuring success: [#6550](https://github.com/desktop/desktop/issues/6550)
 
-#### Support rebase workflows
+#### Support full rebase story, including rebasing one branch onto another locally (in progress)
 
 - Improve rebase workflows: [#5953](https://github.com/desktop/desktop/issues/5953)
 - Measuring success: TBD alongside the work + usability testing
 
-#### Working with uncommitted changes
+#### Branch list grows with merged & deleted branches making it difficult to find those you care about (in progress)
 
-- Improve workflows when you have uncommitted changes: [#6107](https://github.com/desktop/desktop/issues/6107)
-- Measuring success: TBD alongside the work + usability testing
+- Prune branches after they've been deleted: [#750](https://github.com/desktop/desktop/issues/750)
+- Measuring success: N/A for now
```

Comment:
An interesting metric here could simply be "number of branches pruned" within the app - this is scheduled to run daily, applied only to GitHub repositories, and knowing how much cleanup is occurring over time could help gauge success and whether to do it more or less often or for non-GitHub repositories.